ReviewChecklist.md

```diff
@@ -34,6 +38,24 @@ It is also meant to serve as a final checklist for reviewers to go through befor
     - Review updated files before committing them.
         **Are expectations correct and do updated tests still serve their purpose?**
 
+## Abandoned PRs
+- [ ] **Is the submitter still responsive?**
+    - If the PR had no activity from the submitter in the last 2 weeks (despite receiving reviews and our prompts) we consider it abandoned.
+- [ ] **Is the abandoned PR easy to finish or relevant?**
+    - Apply the `takeover` label if the PR can be finished without significant effort or is something that actually needs to be done right now.
+        Otherwise close it.
+        It can still be taken over later or reopened by the sumitter but until then we should not be getting sidetracked by it.
+
+## Light Review
+Before an in-depth review, it is recommended to give new PRs a round of quick, superficial review.
+It should not be an attempt to give complete and detailed feedback but instead give the submitter a
+rough idea if the PR is even on the right track and let them solve the obvious problems on their own.
+
+Light review should focus on three areas:
+- [ ] **Are there any obvious mistakes?** Style issues, bad practices, easy to identify bugs, etc.
+- [ ] **Is there anything missing?** Tests (of the right kind), documentation, etc. Does it address the whole issue?
+- [ ] **Is it the right solution?** Are there better ways to do this? Is the change even necessary?
```

Comment:
Does this sound right?

Reply:
I think it sounds good and covers everything. The only thing I would change is to say
```
Light review should focus on the following three areas in descending order of priority:
```
And then basically reverse the order of your list. For example, if a PR uses a completely wrong approach, or doesn't even address the problem properly, then it's not even worth it to go over style, spelling, etc.

Reply:
Well, for me this is actually the order in which I do things. The last point is the most important but I usually don't know the answer before I'm done reading the PR. On the other hand I can point out style issues and mistakes as I go and doing that costs me almost nothing - it would be more annoying if I actually had to do another pass through the PR just for that.

Reply:
Maybe we should not enforce any order, because people may have different styles/approaches.